src/com/softwaret/kpdf/routing/routes/PublicationRouting.kt

```diff
@@ -32,14 +34,16 @@ fun Routing.publications(controller: PublicationsController) {
             call.respondWith(controller.obtainPublication(Id(getPublicationModel.publicationId)))
         }
 
-        post<PostPublication> { postPublicationLocation ->
+        post<PostPublication> {
```

Comment:
Przy postPublication musimy zwrócić jej id

Reply:
racja

Reply:
Zwracamy cały obiekt publikacji z kontrolera 
```Kotlin
 Response.Created(
                PublicationResponseBody(
                    id,
                    name,
                    author.login,
                    pdf.pdfBase64,
                    metadata.description
                )
            )
```
A samo `postPublicationLocation` to był  uchwyt do url params